examples/Remarketing/UploadCallConversion.php

```diff
@@ -0,0 +1,175 @@
+                PHP_EOL
+            );
+            foreach ($googleAdsException->getGoogleAdsFailure()->getErrors() as $error) {
+                /** @var GoogleAdsError $error */
+                printf(
+                    "\t%s: %s%s",
+                    $error->getErrorCode()->getErrorCode(),
+                    $error->getMessage(),
+                    PHP_EOL
+                );
+            }
+        } catch (ApiException $apiException) {
+            printf(
+                "ApiException was thrown with message '%s'.%s",
+                $apiException->getMessage(),
+                PHP_EOL
+            );
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param int $conversionActionId the ID of the conversion action to upload to
+     * @param string $callerId the caller ID from which this call was placed. Caller ID is expected
+     *     to be in E.164 format with preceding '+' sign. e.g. "+16502531234"
+     * @param string $callStartDateTime the date and time at which the call occurred. The format is
+     *     "yyyy-mm-dd hh:mm:ss+|-hh:mm", e.g. “2019-01-01 12:32:45-08:00”
+     * @param string $conversionDateTime the date and time of the conversion (should be after the
+     *     click time). The format is "yyyy-mm-dd hh:mm:ss+|-hh:mm", e.g.
+     *     “2019-01-01 12:32:45-08:00”
+     * @param float $conversionValue the value of the conversion
+     */
+    public static function runExample(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        int $conversionActionId,
+        string $callerId,
+        string $callStartDateTime,
+        string $conversionDateTime,
+        float $conversionValue
+    ) {
+        // Creates a call conversion by specifying currency as USD.
+        $callConversion = new CallConversion([
+            'conversion_action' => new StringValue([
+                'value' => ResourceNames::forConversionAction($customerId, $conversionActionId)
+            ]),
+            'caller_id' => new StringValue(['value' => $callerId]),
+            'call_start_date_time' => new StringValue(['value' => $callStartDateTime]),
+            'conversion_date_time' => new StringValue(['value' => $conversionDateTime]),
+            'conversion_value' => new DoubleValue(['value' => $conversionValue]),
+            'currency_code' => new StringValue(['value' => 'USD']),
+        ]);
+
+        // Issues a request to upload the call conversion.
+        $conversionUploadServiceClient = $googleAdsClient->getConversionUploadServiceClient();
+        $response = $conversionUploadServiceClient->uploadCallConversions(
+            $customerId,
+            [$callConversion],
+            ['partialFailure' => true]
+        );
+
+        // Prints any partial errors returned.
+        if (!is_null($response->getPartialFailureError())) {
```

Comment:
I would suggest refactoring the "UploadConversionAdjustment.php" and "UploadOfflineConversion.php" to handle partial failure errors in the same approach, using "if else" statement instead of a separate method. 

Perl and Java is following the same approach
https://github.com/googleads/google-ads-java/blob/master/google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UploadOfflineConversion.java#L149

https://github.com/googleads/google-ads-perl/blob/master/examples/remarketing/upload_offline_conversion.pl#L79

Reply:
Why would we want to do that? Could you provide more context? 
(Actually we should discuss separately, as it's not related to this PR. I'm gonna resolve this comment anyway).

Reply:
OK, I will send an email offline.